### PR TITLE
[tasklist] Polish translation (pl-PL)

### PR DESCRIPTION
**Pull request by pithwz**
## Proposed changes
Translate strings from **ENGLISH** to **POLISH**

### CODE REVIEW

**Review by Saibamen — CHANGES_REQUESTED** [review]
Please replace strings instead of duplicating them

**Comment by pithwz** [comment]
> 
> 
> Please replace strings instead of duplicating them

Add suggestions & i apply. Thanks

File: base/applications/cmdutils/tasklist/lang/pl-PL.rc [hunk 1]
```diff
@@ -1,8 +1,12 @@
+/* 
+ *    Polish translation by pithwz - Piotr Hetnarowicz (piotrhwz@gmail.com) (February, 2021)
+ */
 LANGUAGE LANG_POLISH, SUBLANG_DEFAULT
 
 STRINGTABLE
 BEGIN
     IDS_USAGE, "Usage: TASKLIST [/?] [/NH]\n"
+    IDS_USAGE, "Użycie: TASKLIST [/?] [/NH]\n"
```
Comment (Saibamen) [inline base/applications/cmdutils/tasklist/lang/pl-PL.rc:9]:
```suggestion
    IDS_USAGE, "Użycie: TASKLIST [/?] [/NH]\n"
```

File: base/applications/cmdutils/tasklist/lang/pl-PL.rc [hunk 2]
```diff
@@ -13,8 +17,19 @@ Desciption:\
...
+    IDS_INVALID_SYNTAX, "Błąd: Nieprawidłowa składnia.\n"
+    IDS_INVALID_OPTION, "Błąd: Określono nieznaną lub nieprawidłową opcję wiersza polecenia.\n"
+    IDS_ENUM_FAILED, "Błąd: Nie można wyliczyć listy procesów.\n"
+    IDS_OUT_OF_MEMORY, "Błąd: Brak pamięci.\n"
+    IDS_HEADER_IMAGENAME, "Nazwa obrazu"
     IDS_HEADER_PID, "PID"
     IDS_HEADER_SESSION, "Session#"
     IDS_HEADER_MEMUSAGE, "Mem Usage"
+    IDS_HEADER_SESSION, "Nazwa sesji"
+    IDS_HEADER_MEMUSAGE, "Użycie pam."
```
Comment (Saibamen) [inline base/applications/cmdutils/tasklist/lang/pl-PL.rc:33]:
```suggestion
    IDS_HEADER_SESSION, "Numer sesji"
    IDS_HEADER_MEMUSAGE, "Użycie pam."
```

**Review by Saibamen — CHANGES_REQUESTED** [review]
I cannot add comments on lines 9-12 (new: 13-16)

Please commit changes to your branch `patch-47`

File: base/applications/cmdutils/tasklist/lang/pl-PL.rc (hunk 2 above)
Comment (Saibamen) [reply base/applications/cmdutils/tasklist/lang/pl-PL.rc:33]: Also please build ROS with your changes and check if `Numer sesji` is not too long.
Alternative: `Nr sesji`

File: base/applications/cmdutils/tasklist/lang/pl-PL.rc [hunk 3]
```diff
@@ -13,8 +16,17 @@ Desciption:\
     IDS_ENUM_FAILED, "Error: Unable to enumerate the process list.\n"
     IDS_OUT_OF_MEMORY, "Error: Out of memory.\n"
     IDS_HEADER_IMAGENAME, "Image Name"
+Opis:\
```
Comment (HBelusca) [inline base/applications/cmdutils/tasklist/lang/pl-PL.rc:19]: Can you remove all the old english strings please? Thanks.

**Review by HBelusca — CHANGES_REQUESTED** [review]

**Comment by pithwz** [comment]
.

**Pull request closed without merging**